code/frameworks/svelte-vite/src/preset.ts

```diff
@@ -14,14 +14,9 @@ export const viteFinal: NonNullable<StorybookConfig['viteFinal']> = async (confi
   // TODO: set up eslint import to use typescript resolver
   // eslint-disable-next-line import/no-unresolved
   const { svelte, loadSvelteConfig } = await import('@sveltejs/vite-plugin-svelte');
-  const svelteOptions: Record<string, any> = await options.presets.apply(
-    'svelteOptions',
-    {},
-    options
-  );
-  const svelteConfig = { ...(await loadSvelteConfig()), ...svelteOptions };
+  const svelteConfig = await loadSvelteConfig();
 
-  // Add svelte plugin if not present
+  // Add svelte plugin if the user does not have a Vite config of their own
```

Comment:
This comment isn't exactly accurate for right now, though it hopefully will be sometime in the future, right?

Reply:
You said above:

> We don't actually create a file, but we do create a vite config, that we provide to the vite server when it starts up or when we build for production.

I was assuming that this in-memory Vite config would be created whether or not a `vite.config.js` is present. I suppose the old comment might be a bit more accurate in that it applies either way whether or not that file is present. But I think my comment better describes the use case we're trying to address and when we might encounter it, which is perhaps more useful for folks who are editing this code.

Reply:
Sure, I was being pedantic. They could have a vite config without the plugin, and we would add it.  I'm just splitting hairs.